lib/common/schemas.c

```diff
@@ -134,19 +130,27 @@ get_schema_path(const char *name, const char *file)
     return crm_strdup_printf("%s/%s.rng", base, name);
 }
 
+static inline bool
+version_from_filename(const char *filename, int version[2])
```

Comment:
Repeated use of `int[2]` might perhaps justify a typedef'd type.
(put on the line where it became obvious to me)